### DISCUSSION TITLE
Automatic popups / disable popup blocking

### DISCUSSION THREAD

**Discussion by skrecc** [MicrosoftEdge/WebView2Feedback#3918] [Q&A]
Hello I'm using WebView2 for WinForms.

Is possible to disable automatic popup blocking for WebView2 control? To be able show automatic popups. Like in Microsoft Edge. 
![image](https://user-images.githubusercontent.com/31956094/137685467-8fcc56dc-9269-4f11-a36c-805cc074eb46.png)

NewWindowRequested event is fired but the popup open is suppressed.

Thanks
Regards
Petr

**Comment by champnic** [comment]
Hey @skrecc - right now we don't have a way to change this behavior. I'm surprised that NewWindowRequested is being fired but it's not opening the window. Are you doing anything in the event handler?
